runelite-client/src/main/java/net/runelite/client/plugins/virtuallevels/VirtualLevelsPlugin.java

```diff
@@ -102,10 +114,33 @@ public void onScriptCallbackEvent(ScriptCallbackEvent e)
 						continue;
 					}
 
-					level += Experience.getLevelForXp(client.getSkillExperience(s));
+					if (config.alterTotalLevel())
+					{
+						level += Experience.getLevelForXp(client.getSkillExperience(s));
```

Comment:
Store this `skillLevel` var locally since you will use it multiple times afterwards